tests/makefile

```diff
@@ -1,41 +1,321 @@
-INCLUDE		= -I$(EXPORT_ROOT)/include -I/usr/local/include -I../../include
```

Comment:
This is interesting. This makefile is gonna be deprecated. Since we already edit CMakeLists.txt, there shouldn't be any change in this file.